Dockerfile-python3

```diff
@@ -0,0 +1,10 @@
+FROM tuttleofx/tuttleofx-env:python3-latest
```

Comment:
We only provide multiple images for the major versions, so it's easier to maintain like that.